### PR TITLE
list jobs

### LINKED ISSUE #9: Job service
- My job list (auth)
- Job list (public)
- Job detail 
- Register job
- Update job

### PR DESCRIPTION
**Pull request by leandronascimento**
Closes #9

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -86,7 +86,20 @@ export default {
...
+    return {
+      jobs: [],
+    }
+  },
+  mounted() {
+    this.getJobs()
+  },
   methods: {
+    getJobs() {
+      this.$jobRepository.index().then((jobs) => {
```
Comment (carlosxjm) [inline pages/index.vue]: Please, use async/await

File: pages/index.vue [hunk 2]
```diff
@@ -30,8 +30,8 @@
       <section class="recent-jobs">
         <h2 class="section-title">Últimas vagas</h2>
         <ul class="job-list">
-          <li v-for="i in [1, 2, 3, 4, 5, 6]" :key="i" class="job-card-item">
-            <JobCard />
+          <li v-for="job in jobs" :key="job.id" class="job-card-item">
```
Comment (carlosxjm) [inline pages/index.vue]: Show to the user a spinner while the request is loading

Comment (leandronascimento) [reply pages/index.vue]: done!

File: pages/index.vue (hunk 1 above)
Comment (leandronascimento) [reply pages/index.vue]: done.

**Pull request merged**